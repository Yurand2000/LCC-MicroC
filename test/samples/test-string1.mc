char *global_string = "global_string";

int main()
{
    char *local_string = "local_string";

    print_string(global_string);
    print_string(local_string);
    print_string("string_literal");
    return 0;
}