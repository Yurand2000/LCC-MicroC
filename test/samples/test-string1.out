global_string
local_string
string_literal
